packages/main/src/SegmentedButtonItem.ts

```diff
@@ -184,6 +189,10 @@ class SegmentedButtonItem extends UI5Element implements IButton, ISegmentedButto
 	}
 
 	get tabIndexValue() {
+		if (this.disabled) {
```

Comment:
Not sure if we should prevent the tabbing over the disabled segmented items.
If we the first segmented item is disabled and there are two more, which aren't then we'll be unable to tab into them.

Reply:
I think in this case the best option would be to exclude the disabled items from the navigation so the focus will go to the first focusable(enabled) element.